.github/workflows/auto_create_version_bump_pr.yml

```diff
@@ -76,6 +76,13 @@ jobs:
           reviewers: "DataDog/apm-dotnet"
           body: "${{steps.changes.outputs.release_notes}}"
 
+      - name: "Close previous milestone"
+        # We don't close milestones for hotfix releases
+        if: ${{ !contains(github.event.ref, 'hotfix') }}
```

Comment:
We should close milestones for hotfixes, no? 🤔  We have separate milestones for hotfix vs mainline

Reply:
We don't do renaming of vNext milestone for hotfixes, so I copy pasted it. But we don't do version bumps for hotfixes either (see the filter above) so this condition is useless actually. I'll remove it

Reply:
done in 670ff1e7ec78fd6da6c207a9d6798fa99b4f630e